apps/bitwarden/1.32.6-alpine/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       - "${PANEL_APP_PORT_HTTP}:80"
     volumes:
       - ./data:/data
-    image: vaultwarden/server:1.32.6-alpine
+    image: vaultwarden/server:1.32.7-alpine
     labels:  
       createdBy: "Apps"
 networks:  
```

Comment:
The changes you've provided seem to be correct. The only modification between version 1.32.6 and 1.32.7 is the update of the Vaultwarden Docker image tag from `vaultwarden/server:1.32.6-alpine` to `vaultwarden/server:1.32.7-alpine`. This change simply updates the server's software version being used, which should not introduce any issues unless there are breaking changes in this specific release.